src/components/baklava-options-plugin/ColorPickerOption.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="color-picker-container">
+    <div class="__label .text-truncate color-picker-label">{{ name }}</div>
+    <div
+      class="color-picker-box"
+      :style="background"
+      tabindex="-1"
+      @click="startEdit"
+      @focusout="endEdit"
+    >
+      <chrome-picker
+        v-show="isOpen"
+        class="color-picker"
+        :value="displayValue"
+        @input="setColor($event.hex)"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from "vue-property-decorator";
+import { Chrome } from "vue-color";
+
+@Component({
+  components: { "chrome-picker": Chrome },
+})
+export default class ColorPickerOption extends Vue {
+  @Prop({ type: String })
+  name!: string;
+
+  @Prop()
+  // eslint-disable-next-line
+  value!: any; // it has to be any; the baklava source does this
+
+  @Prop({ type: String, default: "#000000" })
+  displayValue!: string;
```

Comment:
Don't need the whole displayValue thing anymore, just use Baklava's `this.value` prop